data/json/items/comestibles/alcohol.json

```diff
@@ -709,6 +709,32 @@
     "fun": 16,
     "freezing_point": 25
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "burdock_wine",
+    "name": { "str": "burdock wine", "str_pl": "burdock wine" },
+    "weight": "35 g",
```

Comment:
35 grams in 250 ml? It's 7 times less denser than water.

Reply:
This is inherited from dandelion wine. I bet there are other examples, but I'll correct that.

Reply:
Looking for wine density, for red merlot as a baseline we have 99g for 100ml, which is alright. If we want to abstract it, we can just simplify the chemical content of our wines to be water + ethanol with a select percentage, like 12.5% ethanol, so you would expect about 97-99g per 100ml depending on ethanol density.

I'm just gonna abstract mine and say "it's 99g per 100ml for burdock wine" and leave everyone else's wines alone until I get permission to grope them.